docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -8,6 +8,12 @@
                  OpenVINO™ toolkit on your system.
 
 
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   NVIDIA GPU <openvino_docs_install_guides_configurations_for_nvidia_gpu>
```

Comment:
I suppose we need to separate NVIDIA and INTEL GPUs and not to reference one to another

Reply:
Have made changes with the https://github.com/openvinotoolkit/openvino/pull/19016/commits/89468d400089ec825a60d511f3042837d189b4c8 commit.